tools/intergen/test/generated_interface_json_tests.cc

```diff
@@ -35,7 +35,7 @@
 #include <test_rpc_interface/interface.h>
 #include <test_rpc_interface/functions.h>
 
-#include "json/reader.h"
+#include "jsoncpp_reader_wrapper.h"
```

Comment:
@ydementieiev I suggest you to specify prefix explicitly like `utils/jsoncpp_reader_wrapper.h` because it is not obvious where this header was taken from. Check this for all occurences of `jsoncpp_reader_wrapper.h` include

Reply:
@AKalinich-Luxoft  done in [da6f3de](https://github.com/smartdevicelink/sdl_core/pull/3155/commits/da6f3de2c9d116ea5264ba91db18c6ba8cfef64e)